src/test/java/br/com/moip/api/CustomerAPITest.java

```diff
@@ -0,0 +1,163 @@
+package br.com.moip.api;
+
+import br.com.moip.request.ApiDateRequest;
+import br.com.moip.request.CreditCardRequest;
+import br.com.moip.request.CustomerRequest;
+import br.com.moip.request.FundingInstrumentRequest;
+import br.com.moip.request.HolderRequest;
+import br.com.moip.request.PhoneRequest;
+import br.com.moip.request.ShippingAddressRequest;
+import br.com.moip.request.TaxDocumentRequest;
+import br.com.moip.resource.Customer;
+import com.rodrigosaito.mockwebserver.player.Play;
+import com.rodrigosaito.mockwebserver.player.Player;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+
+import java.util.Calendar;
+import java.util.GregorianCalendar;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+
+public class CustomerAPITest {
+
+    @Rule
+    public Player player = new Player();
+    private CustomerAPI api;
+
+    @Before
+    public void setUp() {
+        api = new CustomerAPI(new ClientFactory().client(player.getURL("").toString()));
+    }
+
+    @Play("customer/create")
+    @Test
+    public void createCustomer() {
+        CustomerRequest customerRequest = new CustomerRequest()
+                .ownId("own_id")
+                .email("jose_silva0@email.com")
+                .birthdate(new ApiDateRequest()
+                        .date(new GregorianCalendar(1988, Calendar.DECEMBER, 30)
+                                .getTime()))
+                .taxDocument(TaxDocumentRequest.cpf("22222222222"))
+                .phone(new PhoneRequest()
+                        .setNumber("66778899")
+                        .setAreaCode("11"))
+                .shippingAddressRequest(new ShippingAddressRequest()
+                        .city("Sao Paulo")
+                        .complement("8")
+                        .district("Itaim")
+                        .state("SP")
+                        .streetNumber("2927")
+                        .zipCode("01234000")
+                        .country("BRA"));
+
+        Customer customer = api.create(customerRequest);
+
+        assertEquals("CUS-Y6L4AGQN8HKQ", customer.getId());
+        assertEquals("own_id", customer.getOwnId());
+        assertEquals("Jose Silva", customer.getFullname());
```

Comment:
@ratoji vendo bem, como usamos o MockWebServer, o request é inútil. O que importa é o yaml vinculado ao teste.